File: 0001 Code Chef/0015 AC please/implementation/0009 dart/ac_please.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    int? temperature = int.tryParse(stringInput!);
    if(temperature != null){
      if(temperature > 30){
        print("YES");
      }else{
        print("NO");
      }
    }else{
      print("Invlid input");
    }
  }else{
    print("Invalid input");
 }
}

File: 0001 Code Chef/0015 AC please/implementation/0005 kotlin/AcPlease.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var temperature:Int? = stringInput.toIntOrNull();
        if(temperature != null){
            if(temperature > 30){
                println("YES");
            }else{
                println("NO");
            }   
        }
   }else{
        println("Invalid input");
   }
}